pkg/cmd/workspace/start.go

```diff
@@ -63,4 +69,26 @@ var StartCmd = &cobra.Command{
 
 func init() {
 	StartCmd.PersistentFlags().StringVarP(&startProjectFlag, "project", "p", "", "Start a single project in the workspace (project name)")
+	StartCmd.PersistentFlags().BoolVarP(&allFlag, "all", "a", false, "Start all workspaces")
 }
+
+func startAllWorkspaces() {
```

Comment:
Same comment as with `deleteAllWorkspaces`. Return an error and handle it above.